### ISSUE TITLE
Broad knowledge or narrow specialization?

### ISSUE THREAD

**Issue by karataev** [sapegin/ama#2]
Is it necessary for frontend developer to know backend stuff (databases, servers, e.t.c)? Do you read articles or watch videos on technical themes beyond the frontend?

**Comment by sapegin** [comment]
Thanks for you question Eugene!

I think it’s necessary for any web professional to be able to make a site or a web app from start to finish alone. So one need some understanding of frontend, backend, design, typography, writing, devops, etc. I don’t mean you need to know all these things as good as your main specialization but some knowledge definitely helps to do your part of work better and to interact with your colleagues.

Now I’m much more interested in design, typography and writing than in backend or devops. I [read books](http://bibla.ru/sapegin/read/), redesign my own sites every now and then, etc. The problem with backend for me: I don’t really need it anywhere because I do very little of it at work and all my personal sites are static now.

In short: be the best in your specialization but know a little of what’s around you too.

**Comment by karataev** [comment]
Thanks for the detailed response!
